jest/setupTestFramework.js

```diff
@@ -0,0 +1,15 @@
+import path from "path";
+
+if (process.env.TEAMCITY_VERSION != null) {
+  var jasmineReporters = require("jasmine-reporters/src/teamcity_reporter");
+  jasmine.getEnv().addReporter(new jasmineReporters.TeamCityReporter());
+} else if (process.env.JENKINS_VERSION != null) {
```

Comment:
gnah - do we _really_ need this file?

Reply:
We can also do this in each test file, as well but to have this file is definitely the best practice